lib/reporters/cli/index.js

```diff
@@ -175,53 +186,124 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
+            code = res.code,
+            reason = res.reason(),
+            mime = res.contentInfo() || {},
+
+            reqHeadersLen = _.get(req, 'headers.members.length'),
+            resHeadersLen = _.get(res, 'headers.members.length'),
+
+            resTime = util.prettyms(res.responseTime || 0),
+            timings = _.last(_.get(o, 'history.execution.data')),
+            resText = options.verbose ? res.text() : E,
+            reqText = (options.verbose && req.body) ? req.body.toString() : E,
+
+            reqBodyMode = _.get(req, 'body.mode', ''),
+            resSummary = [
+                `${mime.contentType}`,
+                `${mime.mimeType}`,
+                `${mime.mimeFormat}`,
+                `${mime.charset}`
+            ].join(` ${colors.gray(symbols.star)} `);
+
+
+        print.lf(SPC); // also flushes out the circling progress icon
+
+        // for clean readability of code. this section compiles the cli string for one line of
+        // req-res combined summary. this looks somewhat like below:
+        // >> 200 OK ★ 979ms time ★ 270B↑ 793B↓ size ★ 7↑ 7↓ headers ★ 0 cookies
+        print.lf(SPC + SPC + [
+            `${code} ${reason}`,
+            `${resTime} ${colors.gray('time')}`,
+            `${reqSize}${colors.gray(symbols.up)} ${resSize}${colors.gray(symbols.down)} ${colors.gray('size')}`,
+            `${reqHeadersLen}${colors.gray(symbols.up)} ` +
+                `${resHeadersLen}${colors.gray(symbols.down)} ${colors.gray('headers')}`,
+            `${_.get(res, 'cookies.members.length')} ${colors.gray('cookies')}`
+        ].join(` ${colors.gray(symbols.star)} `));
+
+
+        // print the line of response body meta one liner if there is no response body
+        // if there is one, we would print it across the body braces
+        if (!resText) {
+            print.lf(SPC + SPC + resSummary);
+        }
+
+        // print request body
+        if (reqText) {
+            // truncate very large request (is 2048 large enough?)
+            if (reqText.length > 2048) {
+                reqText = reqText.substr(0, 2000) + colors.brightWhite(' (truncated)');
+            }
+
+            reqText = wrap(reqText, `  ${colors.white(symbols.console.middle)} `);
+            print.buffer(`  ${colors.white(symbols.console.top)} ${colors.white(symbols.up)} ${reqBodyMode}\n`,
+                colors.white(`  ${symbols.console.bottom}`))
+                // tweak the message to ensure that its surrounding is not brightly coloured.
+                // also ensure to remove any blank lines generated due to util.inspect
+                .nobuffer(colors.gray(reqText.replace(/\n\s*\n/g, LF) + LF));
+        }
+
+        // print response body
+        if (resText) {
+            print(LF);
+            // truncate very large response (is 2048 large enough?)
+            if (resText.length > 2048) {
+                resText = resText.substr(0, 2000) + colors.brightWhite(' (truncated)');
```

Comment:
too complicated ... it is actually from the CLIUtils wrap function. The truncate message is wrapped and padded AFTER adding this label. If we add truncate later, then there is a chance that it will overflow between the end of message and the lower angle symbol 🤦

Reply:
What if we write `(truncated)` after on a new line always? It won't get wrapped (unless the terminal width is less than ~15 characters).

Reply:
did that exactly! ❤️ 
<img width="565" alt="image" src="https://user-images.githubusercontent.com/232373/129684075-3310ed71-9d7f-483e-a4e4-425e623ad203.png">

Reply:
This is so much cooler! 🚀